libvast/src/execution_node.cpp

```diff
@@ -514,16 +515,21 @@ struct exec_node_state : inbound_state_mixin<Input>,
       schedule_run();
     };
     auto handle_error = [this](caf::error& error) {
+      VAST_DEBUG("{} failed to push", op->name());
       this->current_demand->rp.deliver(std::move(error));
       this->current_demand.reset();
       schedule_run();
     };
     auto response_handle = self->request(
       this->current_demand->sink, caf::infinite, atom::push_v, std::move(lhs));
-    if (force) {
+    if (force or this->outbound_buffer_size >= defaults::max_buffered) {
+      VAST_DEBUG("{} pushes {}/{} buffered elements and suspends execution",
+                 op->name(), capped_demand, this->outbound_buffer_size);
       std::move(response_handle)
         .await(std::move(handle_result), std::move(handle_error));
     } else {
+      VAST_DEBUG("{} pushes {}/{} buffered elements", op->name(), capped_demand,
+                 this->outbound_buffer_size);
```

Comment:
What kind of volume do we expect for these log messages? Is there a danger that it will overwhelm the other logs on a busy node?

Reply:
Once per 250ms or 2^16 elements per operator in a running pipeline, whatever is quickest. Would you prefer this to be a trace log?

Reply:
Based on this description I'd say yes, iiuc this could amount to dozens to hundreds of log lines per seocnd.

Reply:
I moved it to trace.